src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -6,6 +6,8 @@ redirects:
   - /docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-decisions/
   - /docs/new-relic-ai-beta-docs
   - /docs/new-relic-one/use-new-relic-one/new-relic-ai/get-started-incident-intelligence#configure-source-nr-alerts
+  - /docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/
```

Comment:
I'm still not 100% sure on redirects, should the docs.newrelic.com part be here?